### PR TITLE
added new font color

### PR DESCRIPTION
**Pull request by 96RadhikaJadhav**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -17,6 +17,11 @@
 } else {
 	$font = "Open Sans";
 }
+if (isset($_GET['fontColor'])) {
+	$fontColor = $_GET['fontColor'];
+}
+else {
+	$fontColor = "#00796b";
```
Comment (DenverCoder1) [inline index.php:24]: Missing an end `}` after line 24

**Comment by DenverCoder1** [comment]
Hi, looks great! There is a problem with a `}` missing as shown in the comment above. Could you fix that by adding a commit to your branch?

**Comment by DenverCoder1** [comment]
Also it seems you forgot this part:
```
if (fontColor != '') {
	newUrl += '&fontColor=' + encodeURIComponent(fontColor);
}
```

Which should come before "window.location.href = newUrl.replace(/ +/g, '+');" (line 196)

**Comment by 96RadhikaJadhav** [comment]
Done.  From: Jonah LawrenceSent: 11 October 2020 21:23To: DenverCoder1/countdownCc: Radhika; AuthorSubject: Re: [DenverCoder1/countdown] added new font color (#20) Also it seems you forgot this part:if (fontColor != '') {        newUrl += '&fontColor=' + encodeURIComponent(fontColor);}Which should come before "window.location.href = newUrl.replace(/ +/g, '+');" (line 196)—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by DenverCoder1** [comment]
👍 Awesome! Looks good. Merged.

**Comment by 96RadhikaJadhav** [comment]
Hey Jonah, Thanks a lot for your help and support. Thanks,Radhika From: Jonah LawrenceSent: 11 October 2020 21:31To: DenverCoder1/countdownCc: Radhika; AuthorSubject: Re: [DenverCoder1/countdown] added new font color (#20) 👍 Awesome! Looks good. Merged.—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by 96RadhikaJadhav** [comment]
Hi Jonah, I wanted to know if you have any repositories for python. I would like to work in that. Thanks,Radhika From: radhikajadhav014@gmail.comSent: 11 October 2020 22:18To: DenverCoder1/countdown; DenverCoder1/countdownCc: AuthorSubject: RE: [DenverCoder1/countdown] added new font color (#20) Hey Jonah, Thanks a lot for your help and support. Thanks,Radhika From: Jonah LawrenceSent: 11 October 2020 21:31To: DenverCoder1/countdownCc: Radhika; AuthorSubject: Re: [DenverCoder1/countdown] added new font color (#20) 👍 Awesome! Looks good. Merged.—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Pull request merged**